portfolio/src/main/webapp/script.js

```diff
@@ -19,10 +19,15 @@ function addRandomGreeting() {
   const greetings =
       ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
 
+  const visit = 
```

Comment:
Please use both a descriptive name, `visit` doesn't tell me these are places, so maybe `placesToVisit`?

additional nit: This could probably be module wide and not function-scoped, and if you chose to make that switch you should also switch to constant casing per the google JavaScript style guide: https://google.github.io/styleguide/jsguide.html#naming-constant-names